engines/cine/metaengine.cpp

```diff
@@ -245,6 +251,180 @@ SaveStateDescriptor CineMetaEngine::querySaveMetaInfos(const char *target, int s
 	return SaveStateDescriptor();
 }
 
+Common::KeymapArray CineMetaEngine::initKeymaps(const char *target) const {
+	using namespace Common;
+	using namespace Cine;
+
+	Keymap *engineKeyMap = new Keymap(Keymap::kKeymapTypeGame, "cine-default", _("Default keymappings"));
+	Keymap *gameKeyMap = new Keymap(Keymap::kKeymapTypeGame, "game-shortcuts", _("Game keymappings"));
+	Keymap *mouseKeyMap = new Keymap(Keymap::kKeymapTypeGame, "mouse-shortcuts", _("Key to mouse keymappings"));
+	Keymap *introKeyMap = new Keymap(Keymap::kKeymapTypeGame, "intro-shortcuts", _("Exit screen keymappings"));
+
+	Action *act;
+
+	act = new Action(kStandardActionLeftClick, _("Left click"));
```

Comment:
I did not do that because this was maybe the only way the behavior before the keymap and after was exactly replicated, if I mapped it onto StandardAction, it would trigger on instances where it shouldn't like during intro etc..